doc/content/reference/components/network-operations-center.md

```diff
@@ -47,6 +51,35 @@ The following is a brief description of each column in the table:
 
 The data can be sorted by column names and filtered by column values.
 
+### Application Overview
+
+The application overview consists of the following panels:
+
+**Applications active** - a bar chart that visualizes the number of applications with a device that has connected to the network in the selected period, either in the last hour, week, or day.
+
+**Application statistics** - a table layout that displays the **Application ID**, **Active Devices**, and **Average Packet Error Rate** of all the applications on the network in the selected period.
+
+The following is a brief description of each column in the table:
+
+- **Application ID**: displays application IDs of all the applications on the network.
+- **Active Devices**: the number of devices in the application which have uplinked in the **last hour**.
+- **Average Packet Error Rate**: this is calculated by comparing the number of seen frames to the total number of possible frames.
```

Comment:
`in a time frame of one hour. It is an average over all of the devices in the application.`.